readme.md

```diff
@@ -31,32 +33,76 @@ Don't see it? Try reinstalling Package Control. Altenatively, add the repository
 
 If it doesn't work, please file an issue.
 
+If you want contribute or use the latest commits simply git clone the repository `https://github.com/mrjoelkemp/Dependents` to your Packages directory with a directory name of `Dependents`. From sublime choose `Preferences -> Browse Packages`  to easily determine the path to the Packages directory for your install.  Be sure to remove the package via package control first if you had already done so to avoid package control updating the package.
+
+### Configuring settings
+
+There are two ways in which to invoke the package settings.  
+
+One is globally via entires in a user settings file `Preferences -> Package Settings -> Dependents -> Settings-User`.  You can open the `Settings-Default` file and copy and paste the contents as a starter template.   Do not edit the default file as that will be overwritten with updates.
+
+Alternatively you can define settings on a per-project basis by creating
+a `.deprc` file in the root of the sublime project/folder.  The paths you will specify are then **relative** to the sublime project/folder root. This is the preferred way if you are working on more than one project using sublime (aren't we all!)
+
+* `root`: (required) the ultimate/root path at which to limit dependent searching.  When using `.deprc` would be `""` if same as root of sublime project/folder.
+* `config`: (Optional) the path to your requirejs configuration file
+* `sass_root`: (Optional) the path to your Sass files
+* `exclude`: (Optional) a list of paths and/or folder names to exclude from the search for dependents
+ * Omitting folders that contain 3rd party libraries can drastically speed up the search for a large codebase.
+ * The following folders are excluded by default: `node_modules`, `bower_components`, `vendor`
+ * Note: Subdirectories are not supported; you can't supply `some/sub/folder` as an exclusion
+* `build_config`: (Optional) path to your RequireJS Build configuration json file
+ * This should have a "modules" section that lists your bundles (entry points)
+ * Supplying this yields a significant performance speedup when finding relevant app entry points
+* `node_path`: (Optional) path to your node installation. Defaults to `/usr/local/bin` on OSX
+
+example:
+```
+{
+  "root": "public/assets/js",
+  "config": "public/assets/js/config.js",  # Optional
+  "sass_root": "public/assets/sass",       # Optional
+  "exclude": ['jquery.js', 'require.js'],  # Optional
+  "build_config": "public/assets/js/build.json", # Optional
+  "node_path": "/my/node/install/folder"   # Optional
+}
+```
+
+
 ### Usage
 
+There are four ways to trigger the package's commands
+
+* From Main Menu `File -> Dependents`
+* From Context `Right click -> Dependents`
+* Via keyboard [see the key bindings](#key-bindings)
+* Via mouse click [see the mouse bindings](#mouse-bindings)
+
+#### Jump to a dependency
```

Comment:
yes I'm more inclined to have my root parent file open and drill down to
some code I want to edit.  But I see how dependents is "nicer" because in
an open file without good documentation you have no clue what are the
dependent(s).   So putting jump to first is more about the metaphor of
working down from a parent to it's offspring like in a genealogy.

On Wed, Mar 11, 2015 at 7:28 PM, Joel Kemp <notifications@github.com> wrote:

> In readme.md
> <https://github.com/mrjoelkemp/Dependents/pull/131#discussion_r26274247>:
>
> >  ### Usage
> >
> > +There are four ways to trigger the package's commands
> > +
> > +* From Main Menu `File -> Dependents`
> > +* From Context `Right click -> Dependents`
> > +* Via keyboard [see the key bindings](#key-bindings)
> > +* Via mouse click [see the mouse bindings](#mouse-bindings)
> > +
> > +#### Jump to a dependency
>
> Moved it up because that's your favorite feature? :)
>
> I built the Dependents feature first, hence the name of the repo and the
> ordering (not tied to my original ordering at all – I like yours). In this
> order, it reminds me of music albums where the album name has a self-titled
> track somewhere amongst the track list :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mrjoelkemp/Dependents/pull/131/files#r26274247>.
>



-- 
Cheers, Cuidate, TTFN <http://www.youtube.com/watch?v=5Gu50vq5ux4>
David